packages/office-ui-fabric-react/src/components/Shimmer/ShimmerElementsGroup/ShimmerElementsGroup.styles.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  IShimmerElementsGroupStyleProps,
+  IShimmerElementsGroupStyles
+} from './ShimmerElementsGroup.types';
+import { getGlobalClassNames } from '../../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-ShimmerElementsGroup-root'
```

Comment:
Minor nit, but I'd recommend just not having global classNames if you don't need them.

Reply:
We know historically they're problematic. A library (like LPC) uses fabric, and then an app tries to define overrides, thinking that they're the only one that cares about the classname, which breaks the library.

Reply:
@dzearing Is this only about this subcomponent or all of them? I didn't quite understand what you meant by "... if you don't need them.".

Reply:
@dzearing can you elaborate on this one more please?